src/elements/content-sidebar/activity-feed/common/datestamp/__tests__/Datestamp.test.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import Datestamp from '../Datestamp';
+import { MILLISECONDS_PER_YEAR } from '../../../../../../constants';
+import type { Props } from '../Datestamp';
+
+describe('elements/content-sidebar/activity-feed/common/datestamp/Datestamp', () => {
+    const MOCK_DATE = new Date('2021-01-01');
+    const MOCK_DATE_STRING = '2021-01-01T00:00:00+00:00';
+    const MOCK_DATE_TIME = 1609459200000;
+
+    const getDefaults = () => ({ date: MOCK_DATE });
+    const getWrapper = (props: Props) => shallow(<Datestamp {...getDefaults()} {...props} />);
```

Comment:
Seems like the defaults can be inlined here, since the function is unused otherwise?